src/lcd.c

```diff
@@ -43,7 +43,7 @@
 
 
 
-static int row, col;
```

Comment:
Good catch on the size of row/col, the LCD code is reused from another project. For this project, could also use the full 8-bit connection to the LCD on this calculator instead of the 4-bit interface, but I haven't gotten around to it yet.